src/WebUI/src/pages/characters/[id]/index.vue

```diff
@@ -131,10 +132,55 @@ onBeforeRouteUpdate(async to => {
 });
 
 await fetchPageData(character.value.id);
+
+//
+
+const wrapper = ref<ComponentPublicInstance<typeof VOnboardingWrapper> | null>(null);
+const { start, goToStep, finish } = useVOnboarding(wrapper);
+const steps = computed(() => {
+  return [
+    {
+      attachTo: {
+        element: '[data-aq-onboarding-level]',
+      },
+      content: {
+        title: 'Level',
+        description:
+          'Lorem ipsum, dolor sit amet consectetur adipisicing elit. Voluptatem porro repellendus voluptatum ea quis veniam dignissimos quos reiciendis enim non.',
+      },
+    },
+    {
+      attachTo: {
+        element: '[data-aq-onboarding-expMultiplier]',
+      },
+      content: {
+        title: 'Exp. multiplier',
+        description: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Harum, incidunt.',
```

Comment:
Your experience multiplier represents how fast you gain experience as you play cRPG, you can increase this by reaching Level 31 and then "Retiring" - Doing so offers a few benefits, one of which is that it increases your experience multiplier enabling you to gain experience faster.